assets/css/sensei-course-theme/layout.scss

```diff
@@ -112,6 +112,12 @@ body.sensei-course-theme {
 	&__main-content {
 		padding: 32px 0;
 		margin-left: var(--sidebar-width) !important;
+
+		@at-root {
```

Comment:
Sure! Added here: https://github.com/Automattic/sensei/pull/5295/commits/531ed45035476abbf3076e7d7c47e3bd62c59f3b

Reply:
Since it was already approved with the comment change only, I already merged the PR. But if you notice anything else, let me know that I can raise a new PR.